src/sndfile.h.in

```diff
@@ -92,6 +92,10 @@ enum
 	SF_FORMAT_GSM610		= 0x0020,		/* GSM 6.10 encoding. */
 	SF_FORMAT_VOX_ADPCM		= 0x0021,		/* OKI / Dialogix ADPCM */
 
+	SF_FORMAT_NMS_ADPCM_16	= 0x0022,		/* 16kbs NMS G721-variant encoding. */
```

Comment:
Please add spaces so this lines up with the others.

Reply:
@evpobr Cool, didn't know that.

I think that the indentation on src/sndfile.h.in in this pull is correct, but github is displaying it with tabs as 8 spaces, as the .editorconfig doesn't apply to it, what with having a suffix of .h.in rather than .h . Would that be correct?